sleap/gui/dialogs/formbuilder.py

```diff
@@ -224,6 +224,28 @@ def get_results(self) -> Optional[Dict[Text, Any]]:
         return self._results
 
 
+# We are writing this method to override the spinbox in add_item so the increment/decrement is x10/x0.1
+class ExponentialSpinBox(QtWidgets.QDoubleSpinBox):
+    def stepBy(self, steps: int):
+        currValue = self.value()
+
+        if steps == 0:
+            return
+        
+        if steps > 0:
+            newValue = currValue * 10
+            if currValue == 0:
+                newValue = 0.01
+
+        elif steps < 0:
+            newValue = currValue * ((10) ** -1)
+
+        # We need to keep it in the range (0 - 1000)
+        newValue = min(self.maximum(), newValue)  
+        newValue = max(self.minimum(), newValue)
+        self.setValue(newValue)
```

Comment:
I wonder if `setValue` implicitly enforces maximum and minimum for us? It would make sense to me if it did, but can't be too certain...